code-challenges.test.js

```diff
@@ -36,21 +55,67 @@ const fullArr2 = ["hello", 7, 23, -823, false, 78, null, "67", 6, "number"]
 // Expected output: [-823, 7, 23]
 
 
+describe('onlyNums', () => {
+    it('Create a function that takes in an array and returns a new array of only odd numbers sorted from least to greatest.', () => {
+        const fullArr1 = [4, 9, 0, "7", 8, true, "hey", 7, 199, -9, false, "hola"]
+        // Expected output: [-9, 7, 9, 199]
+
+        const fullArr2 = ["hello", 7, 23, -823, false, 78, null, "67", 6, "number"]
+        // Expected output: [-823, 7, 23]
+
+        expect(onlyNums(fullArr1)).toEqual([-9, 7, 9, 199])
+        expect(onlyNums(fullArr2)).toEqual([-823, 7, 23])
+    })
+})
+
+
 // b) Create the function that makes the test pass.
 
+// sort will sort the data but it wont completely work on numbers without the a-b addition (which I have to lookup)
+// create a local variable that filters through the array assessing each value
+// the return needs to be only numbers and (&&) only those numbers that have a remainder after being devided by two
+
+const onlyNums = (array) => {
+    let dataType = array.sort(function (a, b) { return a - b }).filter(value => {
+        return typeof (value) === "number" && (value) % 2 !== 0
+    })
+    return dataType
+}
```

Comment:
Awesome!  Logic looks great here!  You can simplify this even further by instead of assigning line 79 to a variable, you can simply return it on that line allowing you to remove line 82.